content/docs/deploy/core/changelog.mdx

```diff
@@ -6,7 +6,50 @@ lang: en-US
 
 # Core Changelog
 
-Please refer to the [upgrade guide](https://www.pomerium.com/docs/deploy/core/upgrading#since-0190) before upgrading.
+Please refer to the [upgrade guide](/docs/deploy/core/upgrading#0223) before upgrading.
```

Comment:
The addition of the <span>#</span>0223 anchor was reverted, right? Should this be kept the same as it was? Or would it be OK to link to `/docs/deploy/core/upgrading` (no anchor) instead?